lib/components/split-pane.js

```diff
@@ -18,6 +20,28 @@ export default class SplitPane extends PureComponent {
     }
   }
 
+  setupPanes(ev) {
+    this.panes = Array.from(ev.target.parentNode.childNodes);
```

Comment:
(This is not your code originally but..)
Why do not use a local variable ?

Btw, I think it is better to not determine index from DOM. Certainly better to add it to handler when rendering divider.